pages/events/[event].tsx

```diff
@@ -18,6 +16,18 @@ type Event = {
   link: string;
 };
 
+enum DateStatus {
+  past = "past",
+  present = "present",
+  future = "future",
+  loading = "loading",
```

Comment:
```suggestion
  PAST = "past",
  PRESENT = "present",
  FUTURE = "future",
  LOADING = "loading",
```
Enums are usually capitalized by convention